src/scripting/Elsa.Scripting.JavaScript/Options/ScriptOptions.cs

```diff
@@ -2,11 +2,20 @@ namespace Elsa.Scripting.JavaScript.Options
 {
     public class ScriptOptions
     {
-        public ScriptOptions()
-        {
-            AllowClr = true;
```

Comment:
Is defaulting this to false to major of a breaking change? I'm wondering if we should leave these options as opt out (but strongly encouraged to opt out) until the next major release

Reply:
Perhaps not. From the [Jint documentation](https://github.com/sebastienros/jint#accessing-net-assemblies-and-classes) it sounds like we can just turn it off - there's no need to allow access to arbitrary objects from the System namespace. At least, I don't think we're doing that. Of course someone might be doing this, but then they will need to opt-in (and we need to add a note to the next release.

Let's just try it.